src/Behat/Testwork/PathOptions/Printer/ConfigurablePathPrinter.php

```diff
@@ -10,13 +10,16 @@
 
 namespace Behat\Testwork\PathOptions\Printer;
 
+use Symfony\Component\Console\Formatter\OutputFormatter;
+
 final class ConfigurablePathPrinter
 {
     private string $basePath;
 
     public function __construct(
         string $basePath,
         private bool $printAbsolutePaths,
+        private ?string $editorUrl,
```

Comment:
Doesn't this need a default null for BC?

Reply:
updated